examples/prod_failures.rb

```diff
@@ -24,3 +24,4 @@ def main
 end
 
 main if __FILE__ == $PROGRAM_NAME
+aaa    =   "dfdfdfdf"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)